tests/spec/core/dfn-spec.js

```diff
@@ -10,6 +10,9 @@ import {
 
 describe("Core — Definitions", () => {
   afterAll(flushIframes);
+
+  const findDfnErrors = err => err.plugin === "core/dfn";
```

Comment:
I was thinking for something like following, but this is ok too.
```suggestion
  const findDfnErrors = doc => doc.respec.errors.filter(err => err.plugin === "core/dfn");
```